### PR TITLE
Suppress Pandas UserWarning with decorator

### PR DESCRIPTION
**Pull request by nshahquinn**
Bug: T324135

### CODE REVIEW

File: wmfdata/utils.py [hunk 1]
```diff
@@ -41,6 +42,19 @@ def pd_display_all(df):
     ):
         display(df)
 
+def suppress_t324135(fn):
```
Comment (nshahquinn) [inline wmfdata/utils.py]: I would prefer a name that makes sense even if you don't look at the task or the docstring. How about `suppress_pandas_dbapi_warning`?

File: wmfdata/mariadb.py [hunk 2]
```diff
@@ -139,19 +139,12 @@ def run(
         # then turn them into a data frame; this would let us avoid using
         # Pandas's complex SQL machinery.
         for command in commands:
-            try:
```
Comment (nshahquinn) [inline wmfdata/mariadb.py:142]: Looks like you accidentally dropped this `try`.

**Review by nshahquinn — COMMENTED** [review]
I found a couple things for you to revise, but otherwise it looks good! Thank you for taking the time to polish it further :grin:

If you can run the test script (`wmfdata-python/wmfdata_tests/tests.py {{your hive db}}`). If it finishes without any assertion errors, that's a pass.

File: wmfdata/utils.py (hunk 1 above)
Comment (jsnshrmn) [reply wmfdata/utils.py]: Makes sense!

File: wmfdata/mariadb.py (hunk 2 above)
Comment (jsnshrmn) [reply wmfdata/mariadb.py:142]: oopsydoodles; that's what I get for coding when I can't execute. That's addressed now that I have the proxy configured correctly on the stat host. Thanks!

**Comment by jsnshrmn** [comment]
> I found a couple things for you to revise, but otherwise it looks good! Thank you for taking the time to polish it further 😁
> 
> If you can run the test script (`wmfdata-python/wmfdata_tests/tests.py {{your hive db}}`). If it finishes without any assertion errors, that's a pass.

I can't connect to hive yet (I just made a kerberos account request), but I've made the suggested changes.

**Comment by nshahquinn** [comment]
> I can't connect to hive yet (I just made a kerberos account request), but I've made the suggested changes.

Ah, makes sense! I just ran it and everything looks good 🎉

**Pull request merged**